Makefile

```diff
@@ -77,6 +77,9 @@ codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 release: ## Generate release manifests and publish a versioned container image.
 	$(WITH_GOFLAGS) ./hack/release.sh
 
+nightly: toolchain## Generate nightly release manifests and publish a versioned container image.
+	$(WITH_GOFLAGS) ./hack/nightly.sh
```

Comment:
Can we collapse this with the release script?

Reply:
Will do in a follow up PR